src/FSharpLint.Core/Rules/Conventions/UsedUnderscorePrefixedElements.fs

```diff
@@ -0,0 +1,56 @@
+module FSharpLint.Rules.UsedUnderscorePrefixedElements
+
+open System
+open FSharpLint.Framework
+open FSharpLint.Framework.Suggestion
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.Rules
+
+let private checkUsedIdent (previousIdent: Ident) (body: SynExpr) =
+    match body with
+    | SynExpr.Sequential(debugPointAtSequential, isTrueSeq, synExpr, expr2, range) ->
+        match synExpr with
+        | SynExpr.App(exprAtomicFlag, isInfix, funcExpr, argExpr, range) ->
+            match argExpr with
+            | SynExpr.Ident ident ->
+                if previousIdent.idText = ident.idText then
+                    {
+                        Range = range
+                        Message = String.Format(Resources.GetString ("RulesUsedUnderscorePrefixedElements"))
+                        SuggestedFix = None
+                        TypeChecks = []
+                    } |> Array.singleton
+                else
+                    Array.empty
+            | _ ->
+                Array.empty
+        | _ ->
+            Array.empty
+    | _ ->
+            Array.empty  
+
+let runner (args:AstNodeRuleParams) =
+    
+    let error =
+        match args.AstNode with
+        | AstNode.Expression (SynExpr.LetOrUse (isRecursive, isUse, bindings, body, range)) ->
+            match bindings.[0] with
+            | SynBinding(synAccessOption, synBindingKind, mustInline, isMutable, synAttributeLists, preXmlDoc, synValData, headPat, synBindingReturnInfoOption, synExpr, range, debugPointAtBinding) ->
+                match headPat with
```

Comment:
@realmarv what does 'pat' mean? let's not use this abbreviation

Reply:
TBH I don't know, this is how SynBinding is defined. It might be an abbreviation for 'pattern', but not sure